operate/schema/src/main/java/io/camunda/operate/schema/SchemaManager.java

```diff
@@ -62,6 +62,9 @@ public interface SchemaManager {
 
   Map<String, IndexMapping> getIndexMappings(String indexNamePattern);
 
+  /**
+   * @deprecated schema manager is happening in Zeebe exporter now
```

Comment:
🔧 also here. zeebe has several exporters as of this moment. only the CamundaExporter manages the schema